zfs-prune-snapshots

```diff
@@ -353,14 +381,19 @@ for line in "${lines[@]}"; do
 	ht=$(human-time "$delta")
 	hu=$(human-size "$used")
 
+	echo ""
```

Comment:
i don't prefer the extra spacing here, please remove.

like i commented above, since this is a style-only change I'm happy to discuss it in a change where the function of the code isn't being modified.